decamConfig.py

```diff
@@ -0,0 +1,10 @@
+# use "instcal" exposures from the community pipeline until DecamIsrTask is up to snuff
+from lsst.obs.decam.decamNullIsr import DecamNullIsrTask
+config.isr.retarget(DecamNullIsrTask)
+
+from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups
+from lsst.meas.extensions.astrometryNet import LoadAstrometryNetObjectsTask
+config.calibrate.astromRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+config.calibrate.photoRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+
+setPhotocalConfigFromEups(config.calibrate.photoCal)
```

Comment:
`LoadAstrometryNetObjectsTask` is already the default loader, so I suggest removing the 3 relevant lines here (and perhaps in validate_drp as well).

Also I am puzzled by your use of `setPhotocalConfigFromEups`. Is that really wanted? I would expect the code to be far clearer if you just set the appropriate config options for the astrometry catalog you are using. That would also eliminate the need to set the environment variable `ASTROMETRY_NET_DATA_DIR`

Reply:
`setPhotocalConfigFromEups` gets the astrometry reference catalog version and sets the photocal configuration appropriately. It is, unfortunately, currently standard; I believe @parejkoj may fix that though.

Reply:
Based on the responses to RFC-433, I'm not expecting to make any changes to a.net refObjTask. Instead, I plan to replace all my existing a.net refcats with indexed PS1 or Gaia ones, and possibly file another RFC to remove the a.net ref loader.

There is no way that I'm aware of to setup `LoadAstrometryNetObjectsTask` other than the env variables: it doesn't even accept any config options!